src/nestjs-back/src/users/users.service.ts

```diff
@@ -255,6 +255,23 @@ export class UsersService {
     }
   }
 
+  async updateBlockedUsers(id: string, users: User[]) {
+    const blockList: User[] = [];
+
+    for (var toBlock of users) {
+      if (parseInt(id) !== toBlock.id) { /* User can't block themselve */
+        blockList.push(toBlock);
+      }
+    }
+
+    let user = await this.usersRepository.findOne(id, {
+      relations: ['blockedUsers'],
+    });
+
+    user.blockedUsers = blockList;
```

Comment:
Object mutation is possible on a `const` variable. However the variable itself can't be reassigned.

Reply:
Should I use `let`? Since it `blockedUsers` is indeed updated.

Reply:
Yeah my bad. Closing that, will fix later.